data/human/intro missions.txt

```diff
@@ -413,7 +413,100 @@ mission "Intro [2 Interceptor]"
+			`	"This is Captain Marshall," James says as you pull up a chair. "He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
+			choice
+				`	"I'm up for a challenge."`
+					goto mining
+				`	"I'm ready to head out on my own."`
+					decline
+	
+			label "mining"
+			`	"That's good to hear," Captain Marshall says with a smile. "Mining without an asteroid scanner is a test of your piloting skills, but one that pays well.`
+			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. Titanium tends to be a bit larger, with a pitted surface. Lead is light gray and has a boxy shape. Silicon has a bit of a greenish cast and looks like it's been melted in places. I increase the resolution on my control displays to make identification and targeting easier.`
+			`	"Also remember that the more valuable the mineral the longer it takes to mine. Silicon breaks up quickly and you'll fill your hold in a hurry. Titanium is more work, but you make several times the number of credits for the effort."`
+			`	Captain Marshall wishes you luck and heads out into the spaceport. You and James spend a few minutes going over the plan as you finish your drinks. "You should start off easy and mine five tons of silicon. If you've got the hang of it, you can fill your hold with more valuable minerals. There's plenty of both at <destination>. Remember, there's no guaranteed payment in this kind of work; how much you make is based on the value of the minerals in your hold."`
+				accept
+	
+	on visit
+		dialog `You've reached <destination>, but you don't have at least five tons of silicon in your cargo hold. Locate and mine silicon asteroids in this system before returning.`
+	
+	on complete
+		require "Silicon" 5
+		conversation
+			`Once you've landed, you meet up with James and he asks you, "What did you think of mining?"`
+			choice
+				`	"I can't believe anyone would do that much work."`
+				`	"I love flying like that."`
+			`	James nods and says, "Chasing asteroids is always a challenge. But when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly.`
```

Comment:
> `But when you're starting off, mining can be a fast way to build cash for your next ship.`

A while back when I was using a bunch of plugins, I made a lot of money mining platinum/gold/uranium in the "northeast" pirate area. (One plugin was Mega Freight; I was using the Badger I think) (Though I did cheat to get my ship too)